### PR TITLE
Add emmet self closing abbreviation support

### LINKED ISSUE #120203: Emmet suggestion fails to show when user writes self-closing abbreviation
From https://github.com/microsoft/vscode/issues/118904#issuecomment-809808334

1. In an HTML file, type `div/`
2. Notice that suggestions show up to expand `div`, but when `/` is typed, suggestions stop showing up.

Potential code pointers:
- Try adding `/` for relevant languages (everything other than css to stylus) here https://github.com/microsoft/vscode/blob/main/extensions/emmet/src/util.ts#L86
- If that doesn't work, we might need to add `/` to some regexes for https://github.com/microsoft/vscode-emmet-helper/blob/main/src/emmetHelper.ts#L33

### PR DESCRIPTION
**Pull request by sangxxh**
This PR fixes #120203

JSX:

120203_20210401_JSX After

HTML:

120203_20210401_After

### CODE REVIEW

File: extensions/emmet/src/util.ts [hunk 1]
```diff
@@ -84,19 +84,19 @@ export function migrateEmmetExtensionsPath() {
...
+	'xsl': ['!', '.', '}', '*', '$', '/', ']', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9'],
 	'css': [':', '!', '-', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9'],
 	'scss': [':', '!', '-', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9'],
 	'sass': [':', '!', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9'],
 	'less': [':', '!', '-', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9'],
 	'stylus': [':', '!', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9'],
-	'javascriptreact': ['!', '.', '}', '*', '$', ']', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9'],
-	'typescriptreact': ['!', '.', '}', '*', '$', ']', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
+	'javascriptreact': ['!', '.', '}', '*', '$', ']', '/', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9'],
+	'typescriptreact': ['!', '.', '}', '*', '$', ']', '/', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
```
Comment (hantatsang) [inline extensions/emmet/src/util.ts:99]: It looks weird that emmet expands `Component/` to `<Component/>` but not `<Component />`. I found out that that's what emmet does for XML notion though. See https://docs.emmet.io/customization/syntax-profiles/

Comment (rzhao271) [reply extensions/emmet/src/util.ts:99]: Thanks! Would it make more sense to switch over to XHTML expansion, then? I can file a new issue for that.

**Review by rzhao271 — APPROVED** [review]
LGTM, thanks again!

File: extensions/emmet/src/util.ts (hunk 1 above)
Comment (pixieaka) [reply extensions/emmet/src/util.ts:99]: @hantatsang It supposed to be `<Component />` not `<Component/>` 😕

Comment (pixieaka) [reply extensions/emmet/src/util.ts:99]: @rzhao271 Can this change to `<Component />` ?

Comment (hantatsang) [reply extensions/emmet/src/util.ts:99]: @pixieaka It's already been merged so I guess this can be done in a new issue.

**Pull request merged**